react/features/toolbox/components/buttons/VideoQualityButton.web.js

```diff
@@ -0,0 +1,149 @@
+import AKInlineDialog from '@atlaskit/inline-dialog';
```

Comment:
I've made this change locally but I don't like how VideoQualityButton would then depend on toolbox for ToolbarButton and then defaultToolbarButtons would depend on VideoQualityButton. It creates a circular dependency (with no compile problems) between packages. To me it makes sense that buttons remain with where all the other buttons are.